src/components/error-boundary/__tests__/error-boundary.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { act, render, screen } from '@/test-utils/rtl';
+
+import ErrorBoundary from '../error-boundary';
+
+const BadComponent = ({ shouldThrow }: { shouldThrow?: boolean }) => {
+  if (shouldThrow) {
+    throw new Error('test error');
+  }
+
+  return <div>Test component</div>;
+};
+
+const mockError = jest.fn();
+jest.mock('@/utils/logger', () => ({
+  ...jest.requireActual('@/utils/logger'),
```

Comment:
we can remove the require actual here too

Reply:
Oh yes good call, I'll do this